backend-rust/src/graphql_api/block_metrics.rs

```diff
@@ -133,11 +133,11 @@ impl QueryBlockMetrics {
                  p_end.height - p_start.height AS blocks_added,
                  ((p_end.slot_time - p_start.slot_time) /
                      NULLIF(p_end.height - p_start.height, 0)
-                 ) AS avg_block_time,
+                 ) AS avg_block_time_s,
```

Comment:
This is using interval, so there is no need for unit, notice that we later ask for the microseconds of the interval